src/harness/testcases/WINNF_FT_S_FAD_testcase.py

```diff
@@ -285,9 +318,39 @@ def test_WINNF_FT_S_FAD_1(self, config_filename):
       # Register N1 CBSDs.
       request = {'registrationRequest': config['registrationRequests']}
       responses = self._sas.Registration(request)['registrationResponse']
+
       # Check registration responses and get cbsd Id
       self.assertEqual(len(responses), len(config['registrationRequests']))
-      # TODO : to update the test so that it is possible to get a PAL Grant
+      cbsd_ids = []
+      for index, response in enumerate(responses):
+        # in case that SAS reject registration request because the azimuth is not provided
+        if response['response']['responseCode'] != 0:
```

Comment:
Wouldn't this indicate an invalid config? In any case, I don't think we can make this change as it directly contradicts step 2 of the test spec.

Reply:
yes, you are right, I updated the code, but then we are adding an additional constraints for the CBSD config file(to have always the azimuth) which is not necessary true for all the SASes, right?

Reply:
Hm, I guess the right behavior might be to give a warning in this case that the CBSD registration may be rejected, but continue to registration anyway. If we also add a message on line 337 (where we assert that registration was successful) saying something like "For a Cat A device, it is acceptable to reject a registration request which does not include self-consistent azimuth and beamwidth information. If this applies to the rejected registration request, please remove it from the config (or alter the antenna information) and retry."

Reply:
What do you think?

Reply:
but we cann't allow some configs which don't work for all the "correct" SASes, and they depend on a choice in the SAS implemntation, I think we should allow only the configs which work for sure on all the correct SASes, don't you agree?